zephyr/src/utils.c

```diff
@@ -723,53 +717,6 @@ void detect_del_arp_entry(char *ip) {
 /* Environment */
 int service_port = SERVICE_PORT_DEFAULT;
 
-struct interface_info* assign_wireless_interface_info(struct bss_identifier_info *bss) {
```

Comment:
IMHO, we have plan to add AP/P2P support, so, still better if you can get this code to work with any number of interfaces.

Reply:
@krish2718 I have reverted the changes, please have a look and comment.